test/estimation.jl

```diff
@@ -19,3 +19,77 @@ using DSP, Test
     frequencies_estimated = sort(esprit(x, M, p, Fs))
     @test isapprox(frequencies', frequencies_estimated; atol = 1e-2)
 end
+
+@testset "jacobsen" begin
+    # real input: test at two arbitrary frequencies
+    fs = 100
+    t = range(0, 5, step = 1/fs)
+    fr = 28.3
+    sr = cos.(2π*fr*t .+ π/4.2)
+    f_est_real = jacobsen(sr, fs)
+    @test isapprox(f_est_real, fr, atol = 1e-5)
+    fr = 12.45
+    sr = sin.(2π*fr*t .+ 3π/2.2)
+    f_est_real = jacobsen(sr, fs)
+    @test isapprox(f_est_real, fr, atol = 1e-5)
+    # test at higher extreme of DFT
+    fr = 49.9002
+    sr = cos.(2π*fr*t)
+    f_est_real = jacobsen(sr, fs)
+    @test isapprox(f_est_real, fr, atol = 1e-5)
+    # test at lower extreme of DFT
+    @test jacobsen(zeros(10)) == 0.0
```

Comment:
```suggestion
    @test jacobsen(ones(10)) == 0.0
```
to have a well-defined correct result?